CMAKE.md

```diff
@@ -52,10 +57,7 @@ latest version:
     git pull
     git submodule update --init
     cd build
-	cmake .
-    ninja
-    ctest -j 8
-    ninja install
+	cmake .. ninja ctest -j 8 ninja install
```

Comment:
I did:
```
cd ~/src/swipl-devel/build && cmake .. ninja ctest -j 8 ninja install
```
and got:
```
CMake Warning:
  Ignoring extra path from command line:

   "install"


CMake Error: Unknown argument -j
CMake Error: Run 'cmake --help' for all supported options.
```

Reply:
@kamahen , but that does not seem to be the right command. Isn´t that meant to be:

```
cd ~/src/swipl-devel/build && cmake -G ninja .. && ninja && ctest -j 8 && ninja install
```

Reply:
(Yes - I forgot the `cmake -G Ninja ..` because I have the commands in a bash script with `-e`)

Reply:
Apparently @ssardina's edits changes the multi-line command into a single line.  I changed that back in the final edit, so all should be fine with the file as it is now.

Reply:
ouch! sorry about that, that's sloppy. But it is strange, I didn't mean to do that and I dont see see that glitch on my side:

![image](https://github.com/SWI-Prolog/swipl-devel/assets/2368384/5bb3d86e-ea47-4b7c-a5fe-26afac8fcb20)

need to keep an eye on this, quite confused.